### PR TITLE
Enable editing of exporter settings

### PR DESCRIPTION
**Pull request by shokada**

### CODE REVIEW

File: promgen/templates/promgen/exporter_form.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1>Project: {{ project.name }}</h1>
 
 <div class="row">
 
-<form id="exporter_form" action="{% url 'project-exporter' project.id %}" method="post">{% csrf_token %}
+<form id="exporter_form" action="{{ url }}" method="post">{% csrf_token %}
```
Comment (kfdm) [inline promgen/templates/promgen/exporter_form.html]: I think you can just remove `action="{{ url }}"` and let it default to POSTing to the same page. Then you don't need to do extra work in `get_context_data`

File: promgen/views.py [hunk 2]
```diff
@@ -333,6 +333,19 @@ def post(self, request, pk):
...
+    model = models.Exporter
+    button_label = _("Exporter Update")
+    form_class = forms.ExporterForm
+
+    def get_success_url(self):
+        return reverse('project-detail', args=[self.object.project_id])
+
+    def get_context_data(self, **kwargs):
+        context = super(ExporterUpdate, self).get_context_data(**kwargs)
+        context["project"] = self.object.project
```
Comment (kfdm) [inline promgen/views.py]: I think you might be able to replace this using `get_initial`

```
def get_initial(self):
    return {"project": self.object.project}
```

But I need to finish testing myself

File: promgen/templates/promgen/exporter_form.html (hunk 1 above)
Comment (shokada) [reply promgen/templates/promgen/exporter_form.html]: `url` is also used in `ExporterRegister`, so I cannot remove `action="{{ url }}"`
https://github.com/line/promgen/pull/262/files#diff-4148d3344e097bea3c3c8f89325b0f57R629

Comment (kfdm) [reply promgen/templates/promgen/exporter_form.html]: Yes, I think you can remove it from both :)

Comment (shokada) [reply promgen/templates/promgen/exporter_form.html]: I'm sorry. I understood.
fixed https://github.com/line/promgen/pull/262/commits/d39efd4940a5f02f32f9c127a21e44ee3e6f0a76

File: promgen/views.py (hunk 2 above)
Comment (shokada) [reply promgen/views.py]: ```ExporterRegister``` uses ```ProjectMixin``` to set ```project``` in the context.
https://github.com/line/promgen/blob/master/promgen/views.py#L608

Is it better to modify ```ExporterRegister``` to use ```get_initial``` as well?

Comment (kfdm) [reply promgen/views.py]: In that case, can probably use ProjectMixin the same way

Comment (shokada) [reply promgen/views.py]: Since the path of ExporterUpdate  is ```/exporter/<exporter-pk>/edit```, I don't think we can use ProjectMixin in ExporterUpdate

**Pull request closed without merging**